src/actions/simpleActions.js

```diff
@@ -0,0 +1,6 @@
+export const simpleAction = () => dispatch => {
+  dispatch({
+    type: 'SIMPLE_ACTION',
```

Comment:
Ideally we should have a `constant` folder for those to avoid typos but this will do for now